public/app/common/helpers/poll/index.js

```diff
@@ -0,0 +1,43 @@
+function Poll (options) {
+
+  this.options = options;
+  this.options.predicate = this.options.predicate || function truthSayer () { return true; };
```

Comment:
Sorry, I went a little bit functional, where it's okay to use nouns for function names: `mapper`, `reducer` and so forth. I'd argue that you can extend this notion to daily JavaScript :smile_cat: 
You shouldn't use nouns for method names, but I'd say that functions are often more "nouns" than "verbs".
Consider this Promise API [article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) from MDN: the function is called `executor` to give just one example.
```js
new Promise(executor)
```